snapcraft/internal/deltas/_deltas.py

```diff
@@ -0,0 +1,219 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2016 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+import logging
+import os
+import subprocess
+
+
+logger = logging.getLogger(__name__)
+
+
+delta_formats_options = [
+    'xdelta'
+]
+
+
+class DeltaGenerationFailed(Exception):
```

Comment:
Can you put all these exceptions in an `errors` module inside the `deltas` package? And in `__init__.py` import errors.